lib/Data/Validate/Sanctions/Redis.pm

```diff
@@ -69,15 +73,17 @@ sub _load_data {
             $self->{_data}->{$source}->{verified} = $verified // 0;
             $self->{_data}->{$source}->{updated}  = $updated;
             $self->{_data}->{$source}->{error}    = $error // '';
-            $last_time                            = $updated if $updated > $last_time;
+            $last_modification                    = $updated if $updated > $last_modification;
```

Comment:
Lets break down this. 
```
my ($content, $verified, $updated, $error) = $self->{connection}->hmget("SANCTIONS::$source", qw/content verified updated error/)->@*;
```
let's get first only `updated`, if it's not any different from what we have in memory, no point to bother Redis with transferring whole sanctions lists if it's the same as we have  in memore

Reply:
yeah make sense